### PR TITLE
Fix ReferenceError in init-ring-mqtt

### PR DESCRIPTION
**Pull request by brian-mcnamara**
Issue when initializing which results in the error:
```
Saving state file /data/ring-state.json failed with error:
ReferenceError: systemId is not defined
    at main (file:///app/ring-mqtt/init-ring-mqtt.js:83:47)
```

### CODE REVIEW

File: init-ring-mqtt.js [hunk 1]
```diff
@@ -114,4 +114,4 @@ const main = async() => {
     }
 }
 
-main()
\ No newline at end of file
+main()
```
Comment (brian-mcnamara) [inline init-ring-mqtt.js:117]: Github edit, I can add the new line back if desired.

**Comment by tsightler** [comment]
Thanks for the PR.  I actually caught this about a week ago and fixed it in my local instance, but haven't pushed those changes to the dev branch yet.  Since you went through the trouble to submit a PR, I'll merge it.  Thanks!

**Comment by brian-mcnamara** [comment]
Thanks! I figured it was probably known, but it was easy enough to fix and submit.
And thank you for all your work on this project, it's been a great (and easy) integration between ring and HA.

**Comment by Scope666** [comment]
I'll second that, this is one of my favorite integrations!

**Pull request merged**